README.md

```diff
@@ -44,14 +46,6 @@ Create database:
 $ docker-compose up -d
 ```
 
```

Comment:
does a "wait for execution to finish" message referring to "docker-compose up -d" make sense? to prevent someone from running run migrate before docker finishes its tasks